crates/ruff/src/rules/flake8_pyi/rules/simple_defaults.rs

```diff
@@ -221,6 +221,23 @@ fn is_valid_default_value_with_annotation(
     false
 }
 
+pub fn is_type_var_call(checker: &Checker, expr: &Expr) -> bool {
```

Comment:
This is based on a similar helper in pep8-naming https://github.com/charliermarsh/ruff/blob/924bebbb4a0e1a9002a0274cbb58fc85335089dc/crates/ruff/src/rules/pep8_naming/helpers.rs#L57

I considered factoring it out but decided against it for now, I'm not sure they're exactly the same semantically.